tasklist/client/e2e/pages/TestSetupPage.ts

```diff
@@ -16,7 +16,7 @@ class TestSetupPage {
   }
 
   async goToLoginPage() {
-    await this.page.goto('/login');
+    await this.page.goto('./login');
```

Comment:
@vsgoulart playwright [docs](https://playwright.dev/docs/api/class-testoptions#test-options-base-url) says about `baseURL`  configuration:

>When using `page.goto()`, `page.route()`, `page.waitForURL()`, `page.waitForRequest()`, or `page.waitForResponse()` it takes the base URL in consideration by using the `URL()` constructor for building the corresponding URL. Unset by default. Examples:
>	•	`baseURL: http://localhost:3000` and navigating to `/bar.html` results in `http://localhost:3000/bar.html`
>	•	`baseURL: `http://localhost:3000/foo/` and navigating to `./bar.html` results in `http://localhost:3000/foo/bar.html`
>	•	`baseURL: http://localhost:3000/foo` (without trailing slash) and navigating to `./bar.html` results in `http://localhost:3000/bar.html`

In my case, I am using in `playwright.config.ts` `baseURL: http://localhost:8081/tasklist/` which matches the second case. I had to add the trailing slash otherwise `expect(page).toHaveURL` was not working.

So as explained by the doc, the `goto` uses `baseURL`  configuration and not the current page to build the target url

Reply:
@vsgoulart took care of updating the paths :)